test-specifications/crypto-service/accountCreateTransaction.md

```diff
@@ -0,0 +1,439 @@
+| signerKey                 | string | optional          | DER-encoded hex string representation of an additional private key required to sign.                                                        |
+
+### Output Parameters
+
+| Parameter Name | Type   | Required/Optional | Description/Notes                                                                     |
+|----------------|--------|-------------------|---------------------------------------------------------------------------------------|
+| accountId      | string | required          | The ID of the created account.                                                        |
+| status         | string | required          | The status of the submitted `AccountCreateTransaction` (from a `TransactionReceipt`). |
+
+## Property Tests
+
+### **Key:**
+
+- The key for the new account.
+
+| Test no | Name                                                                                          | Input                                   | Expected response                                                              | Implemented (Y/N) |
+|---------|-----------------------------------------------------------------------------------------------|-----------------------------------------|--------------------------------------------------------------------------------|-------------------|
+| 1       | Creates an account with a valid ED25519 public key                                            | key=<VALID_ED25519_PUBLIC_KEY>          | The account creation succeeds.                                                 | N                 |
+| 2       | Creates an account with a valid ECDSAsecp256k1 public key                                     | key=<VALID_ECDSA_SECP256K1_PUBLIC_KEY>  | The account creation succeeds.                                                 | N                 |
+| 3       | Creates an account with a valid ED25519 private key                                           | key=<VALID_ED25519_PRIVATE_KEY>         | The account creation succeeds.                                                 | N                 |
+| 4       | Creates an account with a valid ECDSAsecp256k1 private key                                    | key=<VALID_ECDSA_SECP256K1_PRIVATE_KEY> | The account creation succeeds.                                                 | N                 |
+| 5       | Creates an account with a valid KeyList of ED25519 and ECDSAsecp256k1 private and public keys | key=<VALID_KEYLIST>                     | The account creation succeeds.                                                 | N                 |
+| 6       | Creates an account with a valid KeyList of nested Keylists (three levels)                     | key=<VALID_NESTED_KEYLIST>              | The account creation succeeds.                                                 | N                 |
```

Comment:
Maybe add test with INVALID_NESTED_KEYLIST?

Reply:
How would this be different than test #8? Both would have the TCK send a string of bytes to the server that the SDK wouldn't be able to make sense of

Reply:
Checked one more time and you're right, it won't make any sense then